idp-service/src/test/java/io/mosip/idp/services/ClientManagementServiceTest.java

```diff
@@ -67,7 +67,7 @@ public void createClient_withValidDetail_thenPass() throws Exception {
         ClientDetail entity = new ClientDetail();
         entity.setId("mock_id_v1");
         entity.setStatus("active");
-        Mockito.when(clientDetailRepository.saveAndFlush(Mockito.any(ClientDetail.class))).thenReturn(entity);
+        Mockito.when(clientDetailRepository.save(Mockito.any(ClientDetail.class))).thenReturn(entity);
```

Comment:
what is the reason for changing from saveAndFlush to save ?

Reply:
There was no need for an explicit flush here.